account_invoice_supplier_number_info/view/invoice_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+        <record id="account.invoice_tree" model="ir.ui.view">
+            <field name="mode">primary</field>
+        </record>
+
+        <record id="account.view_account_invoice_filter" model="ir.ui.view">
+            <field name="mode">primary</field>
+        </record>
+
+
+        <record model="ir.ui.view" id="view_account_invoice_supplier_tree">
+            <field name="name">view.account_invoice.supplier.tree</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_tree" />
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='date_invoice']" position="before">
+                    <field name="supplier_invoice_number"/>
+                </xpath>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="view_account_invoice_supplier_search">
+            <field name="name">view.account.invoice.supplier.search</field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.view_account_invoice_filter" />
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="/search/field[@name='number']" position="before">
+                    <field name="supplier_invoice_number"/>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="account.action_invoice_tree2" model="ir.actions.act_window">
+            <field name="search_view_id" ref="view_account_invoice_supplier_search"/>
+        </record>
+
+        <record id="action_invoice_tree2_view1" model="ir.actions.act_window.view">
```

Comment:
Hi @pedrobaeza 
primary function, needs to specific action for each view you want open directly.